worker-workflow/src/main/java/com/github/cafdataprocessing/workflow/WorkflowWorker.java

```diff
@@ -196,6 +199,9 @@ private TransformWorkflowResult transformWorkflow(final WorkflowSpec workflowSpe
         } catch (final WorkflowRetrievalException e) {
             throw new DocumentWorkerTransientException(
                 "Unable to transform workflow. Processing API communication is unhealthy.", e);
+        } catch (final InvalidWorkflowSpecificationException e) {
+            throw new DocumentWorkerTransientException(
+                "Unable to generate workflow. Unable to resolve workflow name to workflow id.", e);
```

Comment:
I don't understand how this is a transient issue.

Reply:
This has been updated to add a failure to the document as it is caused when the workflow name provided cannot be found in the list of workflows available.